src/main/scala/uk/gov/homeoffice/drt/Urls.scala

```diff
@@ -0,0 +1,25 @@
+package uk.gov.homeoffice.drt
+
+import uk.gov.homeoffice.drt.auth.Roles
+import uk.gov.homeoffice.drt.auth.Roles.PortAccess
+
+case class Urls(rootDomain: String, useHttps: Boolean) {
+  val protocol: String = if (useHttps) "https://" else "http://"
+
+  val rootUrl = s"$protocol$rootDomain"
+
+  def portCodeFromUrl(lhrUrl: String): Option[String] = {
```

Comment:
maybe lhrUrl isn't the best name for this?

Reply:
good catch - copy/paste...